src/content/platform-integration/web/wasm.md

```diff
@@ -64,12 +73,12 @@ the `beta` channel (preferred) or `main`.
 
 To learn more about Flutter build release channels and how to switch to
 the `beta` channel, check out the
-[Flutter wiki](https://github.com/flutter/flutter/wiki/Flutter-build-release-channels).
+[Flutter wiki]({{site.github}}/flutter/flutter/wiki/Flutter-build-release-channels).
 
-To then ensure you are running the latest version,
+To ensure you are running the latest version,
 run `flutter upgrade`.
 
-To verify if your Flutter install supports Wasm,
+To verify that your Flutter install supports Wasm,
 run `flutter build web --help`.
 
 At the bottom of the output, you should find experimental Wasm options like:
```

Comment:
**question:** Wouldn't this be conditional on having Wasm support?

```suggestion
If your Flutter install supports Wasm, the response to this command should show experimental Wasm options.
```

Reply:
Hmm, might be good to clarify that, but for now it seems the conditionality is implied by the previous sentence. Feel free to open a new PR with this change as I'd like to get this landed. Thanks!